src/si/jerk.rs

```diff
@@ -0,0 +1,121 @@
+        @inch_per_second_cubed: 2.54_E-2; "in/s³", "inch per second cubed", "inches per second cubed";
+        @kilometer_per_minute_cubed: 4.629_629_629_629_629_E-3; "km/min³",
+            "kilometer per minute cubed", "kilometers per minute cubed";
+    }
+}
+
+#[cfg(test)]
+mod tests {
+    storage_types! {
+        use num::One;
+        use si::quantities::*;
+        use si::jerk as j;
+        use si::length as l;
+        use si::time as t;
+        use tests::Test;
+
+        #[test]
+        fn check_dimension() {
+            let _: Jerk<V> = Length::new::<l::meter>(V::one()) /
+                (Time::new::<t::second>(V::one()) * 
+                Time::new::<t::second>(V::one()) * 
+                Time::new::<t::second>(V::one()));
+        }
+        
+        #[test]
+        fn check_units() {
+            test::<l::yottameter, t::second, j::yottameter_per_second_cubed>();
+            test::<l::zettameter, t::second, j::zettameter_per_second_cubed>();
+            test::<l::exameter, t::second, j::exameter_per_second_cubed>();
+            test::<l::petameter, t::second, j::petameter_per_second_cubed>();
+            test::<l::terameter, t::second, j::terameter_per_second_cubed>();
+            test::<l::gigameter, t::second, j::gigameter_per_second_cubed>();
+            test::<l::megameter, t::second, j::megameter_per_second_cubed>();
+            test::<l::kilometer, t::second, j::kilometer_per_second_cubed>();
+            test::<l::hectometer, t::second, j::hectometer_per_second_cubed>();
+            test::<l::decameter, t::second, j::decameter_per_second_cubed>();
+            test::<l::meter, t::second, j::meter_per_second_cubed>();
+            test::<l::decimeter, t::second, j::decimeter_per_second_cubed>();
+            test::<l::centimeter, t::second, j::centimeter_per_second_cubed>();
+            test::<l::millimeter, t::second, j::millimeter_per_second_cubed>();
+            test::<l::micrometer, t::second, j::micrometer_per_second_cubed>();
+            test::<l::nanometer, t::second, j::nanometer_per_second_cubed>();
+            test::<l::picometer, t::second, j::picometer_per_second_cubed>();
+            test::<l::femtometer, t::second, j::femtometer_per_second_cubed>();
+            test::<l::attometer, t::second, j::attometer_per_second_cubed>();
+            test::<l::zeptometer, t::second, j::zeptometer_per_second_cubed>();
+            test::<l::yoctometer, t::second, j::yoctometer_per_second_cubed>();
+            test::<l::foot, t::second, j::foot_per_second_cubed>();
+            test::<l::inch, t::second, j::inch_per_second_cubed>();
+            test::<l::kilometer, t::minute, j::kilometer_per_minute_cubed>();
+
+            fn test<L: l::Conversion<V>, T: t::Conversion<V>, J: j::Conversion<V>>() {
+                Test::assert_eq(
+                    &Jerk::new::<J>(V::one()),
+                    &(Length::new::<L>(V::one()) /
```

Comment:
Start next line with `/` operator. Do the same for the `*` operators. Tab in the contents of the inner-most `()`s where the time^3 factor is calculated.

```rust
            fn test<L: l::Conversion<V>, T: t::Conversion<V>, J: j::Conversion<V>>() {
                Test::assert_eq(
                    &Jerk::new::<J>(V::one()),
                    &(Length::new::<L>(V::one())
                        / (Time::new::<T>(V::one())
                            * Time::new::<T>(V::one())
                            * Time::new::<T>(V::one()))));
            }
```